tests/testsuite/workspaces.rs

```diff
@@ -2778,9 +2778,9 @@ Available binaries:
     p.cargo("run --bin crate2")
         .with_status(101)
         .with_stderr_data(str![[r#"
-[ERROR] no bin target named `crate2`
+[ERROR] Available bin targets in crate2:
```

Comment:
We are no longer telling the user about the error.  We should preserve the old error message and put this behind a `[HELP]`